app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -1,17 +1,22 @@
-<% unless sender == recipient %>
-  <% follow_request = current_user.sent_follow_requests.find_by(recipient: recipient) %>
+ <div class="<%= dom_id(photo.owner, :follow) %>">
```

Comment:
Instead of this class, could you create an id that includes the sender and recipient id? Afterwards, use this in the create.js.erb and destroy.js.erb to make your changes!